backside/README.md

```diff
@@ -0,0 +1,13 @@
+Setup:
+
+    dep init
+    
+    npm i
+
+    sudo npm i -g serverless-offline
+
+    sudo npm i -g serverless
+
+    make build
+
+    sls offline --useDocker
```

Comment:
Since we're not going to install serverless globally, you can refer to `sls` binary via `./node_modules/.bin/sls offline --useDocker`, however, there is no need to do that. You can simply replace this with `npm start` command, which is defined here: https://github.com/sk8mate/sk8/pull/15/files#diff-f4e58449e174919414e02180f2a1b2cae683df519883ffe1c2d519f9a6017de3R6